Firmware/mainfirmware/mainfirmware.ino

```diff
@@ -51,6 +53,11 @@ Controller controle_esq = Controller(KC_ESQ, TI_ESQ, TD_ESQ, TS);
 Controller controle_dir = Controller(KC_DIR, TI_DIR, TD_DIR, TS);
 Controller directionController = Controller(0.1, 0, 0, 1/FREQ_ACT);
 
+// Communication
+
+Exposer* exposer = &Exposer::self();
+
+
```

Comment:
Try to no add to much space

// Communication
Exposer* exposer = &Exposer::self();